packages/date-picker/docs/controls/validationControls.ts

```diff
@@ -0,0 +1,76 @@
+export const validationControls = {
```

Comment:
Any possibility of making a custom control which is the combination of the valid combinations instead? And disable the controls for the individual props.